[
  "bank_account_test.py::BankAccountTest::test_checking_balance_of_closed_account_throws_error"
]